src/services/UserProfileService.ts

```diff
@@ -0,0 +1,33 @@
+import { auth } from '@libs/firebase-sdk/firebase-sdk';
```

Comment:
These are API, not services. Other services are backend-only code.

Reply:
Well... currently we have services running on the frontend only (e.g.: `AuthService.ts`). We can change these to API like `AuthApi.ts` but keep in mind that those code doesn't just call our API, it calls Firebase's API as well.